src/routes/blog/spyder-6-daniel/+page.md

```diff
@@ -0,0 +1,102 @@
+---
+
+[Daniel Althviz](https://github.com/dalthviz) is one of our longest-tenured Spyder developers, having started working on the project near the end of 2016.
+Now eight years later, he's the project's co-maintainer and its our release manager, responsible for the many steps required to release Spyder across all our various platforms.
+On top of that, he is also the lead maintainer of two of the most important libraries on which Spyder, and a lot of other software depends: the [QtPy](https://github.com/spyder-ide/qtpy) GUI abstraction layer and the [QtAwesome](https://github.com/spyder-ide/qtawesome) icon font framework.
+
+In 2023, Spyder received a [Chan Zuckerberg Initiative](https://chanzuckerberg.com/) Essential Open Source Software for Science [Cycle 5 grant, number 384](https://chanzuckerberg.com/eoss/proposals/enhancing-spyder-ide-remote-support-for-scientific-research-in-python/) to support a new remote development architecture and features in Spyder and related development.
+This has helped us fund Daniel's work on many of his contributions to Spyder 6 connected to that!
+
+Thanks to all of that, Daniel has put in a lot of work toward Spyder 6's development.
+Read on to hear the highlights!<br><br>
+
+
+![The Spyder Contributions page on GitHub, showing a graph of all contributions over time, and Daniel highlighted as the number two contributor between Feb 1 2023 and the present, after lead maintainer Carlos Cordoba](daniel-contributions.png "Daniel has the second-most contributions to the core Spyder repository since Feb 2023")
+
+## Spyder Editor migration to new plugin API
+
+We introduced a brand new plugin API with Spyder 5, and as part of that refactored most of Spyder's panes and many other of its internal components to be plugins that use it, rather than tightly coupled with Spyder itself.
+That both makes Spyder much more modular and easier to maintain, but also gives external plugins all the same power and flexibility that those Spyder internal panes and components have!
+However, the one major exception to that in Spyder 5 was the Editor, which wasn't migrated yet due to the high level of complexity and the further new features that would need to be added to the plugin API.
+
+Migrating the Editor to use the modular plugin API was the final piece of the puzzle to complete the API migration making Spyder a fully modular, pluggable application, avoid having to maintain two duplicate APIs and allow interoperability with other plugins, such as those being developed for remote development in Spyder 6.
+Therefore, along with its related API additions and improvements, it also was a prerequisite for fully implementing our new remote development architecture.
+Therefore, we prioritized it for Spyder 6, with Daniel taking the lead on implementing it!
```

Comment:
I think this sounds kind of like nothing was done up to the point I got in charge of the Editor migration work. However, I would say that a lot of work was done for people like Edgar and Stephanie and even Gonzalo. I would say here something like:

```suggestion
Therefore, we prioritized it for Spyder 6, with Daniel pushing forward an full migrated Editor implementation!
```

While also adding to the context presented before some of the work done that lead to be able to do the work I did for the Editor migration like:

* https://github.com/spyder-ide/spyder/pull/13440
* https://github.com/spyder-ide/spyder/pull/13436
* https://github.com/spyder-ide/spyder/pull/17467